controller/scheduler-v2/scheduler.go

```diff
@@ -568,33 +554,41 @@ func (s *Scheduler) sendDiffRequests(f *Formation, diff Processes) {
 }
 
 func (s *Scheduler) followHost(h utils.HostClient) {
-	_, ok := s.hostStreams[h.ID()]
-	if ok {
+	if _, ok := s.hostStreams[h.ID()]; ok {
 		return
 	}
 
 	log := fnLogger("host.id", h.ID())
 	log.Info("streaming job events")
-	// TODO: reconnect this stream, stopping only if unfollowHost is called
 	events := make(chan *host.Event)
 	stream, err := h.StreamEvents("all", events)
 	if err != nil {
 		log.Error("error streaming job events", "err", err)
 		return
 	}
+
+	log.Info("getting active jobs")
+	jobs, err := h.ListJobs()
+	if err != nil {
+		log.Error("error getting active jobs", "err", err)
+		return
+	}
+	log.Info(fmt.Sprintf("got %d active job(s) for host %s", len(jobs), h.ID()))
+
+	for _, job := range jobs {
+		s.handleActiveJob(&job)
```

Comment:
`StreamEvents` does not return the current state like other streams do, so this makes sure we have the current state when we start following a host (rather than relying on the next trigger of `SyncJobs`).